packages/sources/mycryptoapi/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,92 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError, assertSuccess } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+
+  describe('successful calls @integration', () => {
```

Comment:
```suggestion
  describe('successful calls, () => {
```
We can remove the `@integration` identifier, it's no longer needed